src/driver/amdxdna/ve2_mgmt.c

```diff
@@ -214,17 +215,21 @@ int ve2_mgmt_destroy_partition(struct amdxdna_ctx *hwctx)
 {
 	struct amdxdna_dev *xdna = hwctx->client->xdna;
 	struct amdxdna_ctx_priv *priv = hwctx->priv;
+	u32 start_col = priv->start_col;
+	u32 num_col = priv->num_col;
+	u32 rel_col;
 	int ret;
 
 	if (!priv->aie_part) {
 		XDNA_ERR(xdna, "Parition does not have aie device handle");
 		return -ENODEV;
 	}
 
-	for (u32 col = 0; col < priv->num_col; col++) {
-		ret = cert_clear_partition(xdna, priv->aie_part, col);
+	for (u32 col = start_col; col < start_col + num_col; col++) {
```

Comment:
what is the change? earlier loop looks simpler.

Reply:
updated the logic, please review.